pom.xml

```diff
@@ -6,7 +6,7 @@
     <groupId>io.github.deweyjose</groupId>
     <artifactId>graphqlcodegen-maven-plugin</artifactId>
     <packaging>maven-plugin</packaging>
-    <version>1.51.3</version>
+    <version>1.51.4</version>
```

Comment:
1.60.0

Reply:
Thanks for reviewing this PR.  Updated my PR with the suggested version number